src/datatypes/src/value.rs

```diff
@@ -1090,12 +1101,44 @@ impl<'a> PartialOrd for ListValueRef<'a> {
     }
 }
 
+impl<'a> ValueRef<'a> {
+    pub fn byte_size(&self) -> usize {
```

Comment:
What about `data_size()`? It only counts the size of the data. We also need some documents about this method (The size is estimated and only considers the data size).
```suggestion
    pub fn data_size(&self) -> usize {
```

Reply:
Could we name it `estimated_size`?

Reply:
`estimated_data_size`? Since we only consider internal data.